cafes/views.py

```diff
@@ -44,6 +47,23 @@ def delete(self, request, cafe_id):
         review.delete()
         return JsonResponse({'MESSAGE' : 'REVIEW_DELETED'}, status=204)
     
+    def get(self, request, cafe_id):
+        reviews = Review.objects.filter(cafe_id = cafe_id, comment_on_review_id__isnull = True)
+
+        review_list = [{   
+                'id'                : review.user_id,
+                'nickname'          : User.objects.get(id=review.user_id).nickname,
+                'profile_image'     : User.objects.get(id=review.user_id).image_url,
+                'star_rating'       : StarRating.objects.get(
+                    cafe_id = cafe_id, user_id = review.user_id
+                    ).score,
+                'content'           : review.content,
+                'review_like'       : ReviewLike.objects.filter(review_id=review.id).count(),
+                'comment_on_review' : Review.objects.filter(comment_on_review_id = review.id).count()
+                } for review in reviews
+            ]
```

Comment:
`User.objects.get(id=review.user_id).nickname`, 
`User.objects.get(id=review.user_id).image_url`,
`StarRating.objects.get(cafe_id = cafe_id, user_id = review.user_id).score` .. 이런 코드 부분은
review부터 접근해서 가져오도록 수정해주세요~

```suggestion
        review_list = [{   
                'id'                : review.user_id,
                'nickname'          : review..
                'profile_image'     : review..
                'star_rating'       : review..,
                'content'           : review.content,
                'review_like'       : review..,
                'comment_on_review' : review..
                } for review in reviews
            ]
```